modules/apps/transfer/keeper/relay_forwarding_test.go

```diff
@@ -561,6 +561,10 @@ func (suite *KeeperTestSuite) TestAcknowledgementFailureScenario5Forwarding() {
 	err = path1.EndpointB.AcknowledgePacket(packetFromBtoA, errorAckOnA.Acknowledgement())
 	suite.Require().NoError(err)
 
+	// Check that B deleted the forwarded packet.
```

Comment:
We can probably add this check to some other tests as well (like the happy path).

Reply:
Good idea, done!